.github/ISSUE_TEMPLATE/docs-update-issue.md

```diff
@@ -2,21 +2,37 @@
 name: Docs update issue
 about: Request an update to a doc
 title: ''
-labels: ''
+labels: 'Bug'
 assignees: saudsami, atovpeko
 
 ---
 
-**Business case**: How will this issue help our customers?
+**SDK/Product**
+
+<!-- Signaling, Broadcast Streaming, Voice... -->
```

Comment:
Hmmmmmm, by putting things all in comments it will take ages to make a lot of tickets. Could be better to have things to remove.